operators/api/v1alpha2/instance_types.go

```diff
@@ -92,6 +92,18 @@ type InstanceSpec struct {
 	CustomizationUrls *InstanceCustomizationUrls `json:"customizationUrls,omitempty"`
 }
 
+// InstanceAutomationStatus reflects the status of the instance's automation (termination and submission).
+type InstanceAutomationStatus struct {
+	// The last time the Instance desired status was checkedd.
+	LastCheckTime metav1.Time `json:"lastCheckTime"`
+
+	// The termination time of the Instance.
+	TerminationTime metav1.Time `json:"terminationTime"`
```

Comment:
Actually I'd set this field only upon effective termination, not really putting the expected time from the response of the statusCheck

Reply:
I personally feel it may be useful to set it in response to the statusCheck, to have a visual idea of when the instances will e terminated (without having to dig into the logs)